source/MRMesh/MRSceneRoot.h

```diff
@@ -13,10 +13,21 @@ class SceneRoot
     MRMESH_API static Object& get();
     MRMESH_API static std::shared_ptr<Object>& getSharedPtr();
 
+    MRMESH_API static bool isScenePathEmpty();
+    MRMESH_API static void clearScenePath();
+
+    MRMESH_API static std::filesystem::path getScenePath();
+    MRMESH_API static std::shared_ptr<std::filesystem::path>& getScenePathSharedPtr();
+
+    MRMESH_API static std::string getSceneFileName();
+
+    MRMESH_API static void setScenePath( const std::filesystem::path& scenePath );
```

Comment:
getter and setter should be enough, and please provide comment about what this path represents